daisy_workflows/image_build/enterprise_linux/kickstart/rocky_linux_optimized_gcp_8.cfg

```diff
@@ -6,7 +6,7 @@ text --non-interactive
 url --url="https://dl.rockylinux.org/pub/rocky/8/BaseOS/$arch/os"
 repo --name=AppStream --baseurl="https://dl.rockylinux.org/pub/rocky/8/AppStream/$arch/os"
 repo --name=PowerTools --baseurl="https://dl.rockylinux.org/pub/rocky/8/PowerTools/$arch/os"
-repo --name=CloudSig --baseurl="https://dl.rockylinux.org/pub/sig/8/cloud/$arch" --install
+repo --name=CloudSig --baseurl="https://dl.rockylinux.org/pub/sig/8/cloud/$arch"
```

Comment:
aiui this should be $basearch not $arch

Reply:
Oh yeah- weird- it worked anyway. But changed it.